src/App.js

```diff
@@ -1,30 +1,27 @@
 import { useContext } from 'react';
-import CardCtx from './context/card-context.js';
-import CardList from './Card/CardList';
-import AddCard from './Card/AddCard';
-import { VscTrash } from 'react-icons/vsc';
+import { Routes, Route, NavLink, Navigate } from 'react-router-dom';
+import Cards from './pages/Cards';
+import PageNotFound from './pages/PageNotFound';
+import SignIn from './pages/SignIn';
+import UserCtx from './context/user-context';
 import './index.css';
-import Badge from 'react-bootstrap/Badge';
-import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 
 const App = () => {
-// /{cardCtx.error && <h2>{cardCtx.error}</h2>} 
-  const cardCtx = useContext(CardCtx);
+  const userCtx = useContext(UserCtx);
+  
   return (
     <div>
-      <h1 className="header">
-        Cards
-        <Badge className="badge bg-secondary">{cardCtx.getCount()}</Badge>
-      </h1>
-       
-      {cardCtx.show && <AddCard />}
-      <h3><button type="button" className="tab" onClick={cardCtx.showModal}>
-        <b>Добавить новую карточку</b>
-      </button></h3>
-      <VscTrash className="icon_right" onClick={cardCtx.onDeleteSelected} />
-      <label className="icon_left" > <input type="checkbox" id="card_onlyviewmode" onClick={cardCtx.setViewModeOnlyHandler} />Только просмотр</label>
-      <CardList />
-    </div>
+      <NavLink className={(navData) => navData.isActive ? 'active' : ''} to='home' >Home</NavLink>
+      <NavLink className={(navData) => navData.isActive ? 'active' : ''} to='signin' >Sign In</NavLink>
```

Comment:
A bit of weird case.

![image](https://user-images.githubusercontent.com/52723192/173819129-f30db0b5-cc8e-4df9-9e3e-9a959f6eca21.png)

1. No space found between links.
2. Sign in link is available even when user 'is signed'.

Reply:
I know that the task haven't supposed the actual authorization. Still, since you've implemented it, need to leave comments.